src/Symfony/Component/Notifier/Bridge/Telegram/TelegramTransport.php

```diff
@@ -80,7 +80,7 @@ protected function doSend(MessageInterface $message): void
         if (200 !== $response->getStatusCode()) {
             $result = $response->toArray(false);
 
-            throw new TransportException(sprintf('Unable to post the Telegram message: %s (%s).', $result['description'], $result['error_code']), $response);
+            throw new TransportException(sprintf('Unable to post the Telegram message: %s (%s).', $result['description'] ?? '', $result['error_code'] ?? ''), $response);
```

Comment:
Added because an empty response would otherwise throw an Undefined index exception

Reply:
In which case do we get an empty response?

Reply:
I thought maybe if the server was behaving unexpectedly / is having issues. But maybe that's a bit of an edge case.

Reply:
We should not. The contract for Telegram is to make this information available. So, I would revert this change.

Reply:
Np, reverted.